tests/test.hs

```diff
@@ -0,0 +1,1086 @@
+{-# LANGUAGE CPP  #-}
```

Comment:
This file was removed from `develop`. Likely, it needs to be removed from this PR as well.